test failure on MacOS

```
py: commands[1]> coverage run tests/aggregate_tests.py
.............................................Assertion failed: (ret == 0), function digest_final, file boringssl_crypto_digests.m, line 41.
py: exit -6 (0.86 seconds) /Users/runner/work/securesystemslib/securesystemslib> coverage run tests/aggregate_tests.py pid=4425
py: FAIL code -6 (53.39=setup[52.46]+cmd[0.08,0.86] seconds)
  evaluation failed :( (53.44 seconds)
##[error]Process completed with exit code 250.
```

This has started spontaneously happening on MacOS tests
* possibly a MacOS version upgrade at GitHub
* unsure why the test output is so useless -- the dots imply some tests ran successfully but no details are printed

--- a/.github/workflows/_test.yml
+++ b/.github/workflows/_test.yml
@@ -20,19 +20,19 @@ jobs:
           - python-version: "3.10"
             os: macos-latest
             toxenv: py
-          - python-version: "3.13"
+          - python-version: "3.14"
             os: macos-latest
             toxenv: py
-          - python-version: "3.13"
+          - python-version: "3.14"
             os: windows-latest
             toxenv: py
-          - python-version: "3.13"
+          - python-version: "3.14"
             os: ubuntu-latest
             toxenv: purepy
-          - python-version: "3.13"
+          - python-version: "3.14"
             os: ubuntu-latest
             toxenv: py-no-gpg
-          - python-version: "3.13"
+          - python-version: "3.14"
             os: ubuntu-latest
             toxenv: py-test-gpg-fails
           - python-version: "3.10"
@@ -69,11 +69,8 @@ jobs:
             echo "PYKCS11LIB=/usr/lib/softhsm/libsofthsm2.so" >> $GITHUB_ENV
 
           elif [ "$RUNNER_OS" == "macOS" ]; then
-            ## disabled due to https://github.com/secure-systems-lab/securesystemslib/issues/1015
-            unset PYKCS11LIB
-            echo "Skipping HSM tests on MacOS"
-            # brew install softhsm
-            # echo "PYKCS11LIB=$(brew --prefix softhsm)/lib/softhsm/libsofthsm2.so" >> $GITHUB_ENV
+            brew install softhsm
+            echo "PYKCS11LIB=$(brew --prefix softhsm)/lib/softhsm/libsofthsm2.so" >> $GITHUB_ENV
           elif [ "$RUNNER_OS" == "Windows" ]; then
             echo "Skipping HSM tests on Windows"
             # see https://github.com/secure-systems-lab/securesystemslib/issues/520

